R/utilities.R

```diff
@@ -1148,3 +1148,185 @@ ToNumeric <- function(x){
   }
   return(x)
 }
+
+#' Export Seurat object for UCSC cell browser
+#'
+#' @param object Seurat object
+#' @param dir output directory path
+#' @param dataset.name name of the dataset
+#' @param meta.fields vector of metadata fields to export
+#' @param meta.fields.names list that defines metadata field names
+#'                          after the export. Should map metadata
+#'                          column name to export name
+#' @param embeddings vector of embedding names to export
+#' @param markers.file path to file with marker genes
+#' @param cluster.field name of the metadata field containing cell cluster
+#' @param cb.dir in which dir to create UCSC cell browser content root
+#' @param port \emph{experimental} on which port to run UCSC cell browser after export
+#'
+#' @export
+#'
+#' @examples
+#' ExportToCellbrowser(object = pbmc_small, dataset.name = "PBMC", dir = "out")
```

Comment:
For examples that require python packages, we generally recommend wrapping them in `/dontrun{}` so that the CI/CRAN checks will pass.